src/server/json_family.cc

```diff
@@ -884,33 +884,50 @@ OpResult<vector<OptLong>> OpArrIndex(const OpArgs& op_args, string_view key,
 }
 
 // Returns string vector that represents the query result of each supplied key.
-OpResult<vector<OptString>> OpMGet(const OpArgs& op_args, const vector<string_view>& keys,
-                                   JsonExpression expression) {
-  vector<OptString> vec;
-  for (auto& it : keys) {
-    // OpResult<JsonType> result = GetJson(op_args, it);
-    OpResult<JsonType*> result = GetJson(op_args, it);
-    if (!result) {
-      vec.emplace_back();
+vector<OptString> OpMGet(JsonExpression& expression, const Transaction* t, EngineShard* shard) {
```

Comment:
why `expression` is passed by mutable reference, @iko1 ?
the reason I am asking is if it's been mutated then there is a potential data-race since `OpMGet` is called in parallel from multiple threads.

Reply:
so either lets change it to const ref or pass it by value